__tests__/app.test.js

```diff
@@ -206,3 +205,32 @@ describe("Post /api/articles/:article_id/comments", () => {
     await request(app).patch("/api/articles/nonsense").send(patch).expect(400);
   });
 });
+describe("DELETE comment by ID", () => {
+  test("responds with a 404 if no comment id is provided", async () => {
+    const response = await request(app).delete("/api/comments/").expect(404);
+  });
+  test("responds with an empty object if there's no content after deleting", async () => {
```

Comment:
I'd just state the status code in your description here, to be consistent with the rest of your tests 👍